engine/src/test/java/io/zeebe/engine/processing/workflowinstance/CreateWorkflowInstanceProcessorTest.java

```diff
@@ -1,338 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Zeebe Community License 1.0. You may not use this file
- * except in compliance with the Zeebe Community License 1.0.
- */
-package io.zeebe.engine.processing.workflowinstance;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.entry;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-import io.zeebe.el.ExpressionLanguageFactory;
-import io.zeebe.engine.processing.CommandProcessorTestCase;
-import io.zeebe.engine.processing.common.ExpressionProcessor;
-import io.zeebe.engine.processing.common.ExpressionProcessor.VariablesLookup;
-import io.zeebe.engine.processing.deployment.transform.DeploymentTransformer;
-import io.zeebe.engine.processing.streamprocessor.TypedRecord;
-import io.zeebe.engine.state.KeyGenerator;
-import io.zeebe.engine.state.ZeebeState;
-import io.zeebe.engine.state.deployment.DeployedWorkflow;
-import io.zeebe.engine.state.instance.ElementInstance;
-import io.zeebe.engine.state.mutable.MutableElementInstanceState;
-import io.zeebe.engine.state.mutable.MutableVariableState;
-import io.zeebe.engine.state.mutable.MutableWorkflowState;
-import io.zeebe.model.bpmn.Bpmn;
-import io.zeebe.model.bpmn.BpmnModelInstance;
-import io.zeebe.protocol.impl.record.value.deployment.DeploymentRecord;
-import io.zeebe.protocol.impl.record.value.deployment.Workflow;
-import io.zeebe.protocol.impl.record.value.workflowinstance.WorkflowInstanceCreationRecord;
-import io.zeebe.protocol.impl.record.value.workflowinstance.WorkflowInstanceRecord;
-import io.zeebe.protocol.record.RejectionType;
-import io.zeebe.protocol.record.intent.WorkflowInstanceCreationIntent;
-import io.zeebe.protocol.record.intent.WorkflowInstanceIntent;
-import io.zeebe.protocol.record.value.BpmnElementType;
-import io.zeebe.test.util.MsgPackUtil;
-import io.zeebe.test.util.collection.Maps;
-import io.zeebe.util.buffer.BufferUtil;
-import java.io.ByteArrayOutputStream;
-import java.util.Map;
-import org.agrona.DirectBuffer;
-import org.agrona.MutableDirectBuffer;
-import org.agrona.concurrent.UnsafeBuffer;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.junit.MockitoJUnitRunner;
-
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
-public final class CreateWorkflowInstanceProcessorTest
-    extends CommandProcessorTestCase<WorkflowInstanceCreationRecord> {
```

Comment:
@saig0 @Zelldon Why did this PR delete this test case? Was this on purpose?

Reply:
It was easy to make these test cases pass again, so it's reintroduced in https://github.com/zeebe-io/zeebe/pull/6282

Reply:
It was on purpose yes, because they are ugly and hard to maintain.

Reply:
Ok, then I remove it again from my PR